src/errors.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+/**
+ * Exception raised when trying to revert migration that is not possible
+ * to revert.
+ */
```

Comment:
For these errors can we use an [error code](https://github.com/IndigoUnited/js-err-code#readme) instead of a name, and add tests that cause these errors to be thrown and check the error code. Here's an [example of a test that checks an error code](https://github.com/ipfs/js-datastore-s3/blob/master/test/index.spec.js#L163)